### PR TITLE
IDEM-9

### PR DESCRIPTION
**Pull request by pondpiu**
#9 

verify pgp via idem proxy & user email

### CODE REVIEW

File: app/src/hooks/usePpg.ts [hunk 1]
```diff
@@ -82,68 +86,122 @@ const usePgp = (): Hooks => {
...
+  //         );
+  //       }
+  //     }
+  //   } catch (error: any) {
+  //     Alert.alert(
+  //       AlertTitle.Error,
+  //       "Could not verify email, please check your email and try again"
+  //     );
+  //   }
+  // };
```
Comment (rellfy) [inline app/src/hooks/usePpg.ts:145]: I think this should be deleted and not commented out.

File: app/src/hooks/usePpg.ts [hunk 2]
```diff
@@ -16,11 +17,16 @@ type Hooks = {
   generateKeyPairFromPrivateKey: (
     privateKey: string | undefined
   ) => Promise<void>;
-  publishPGPPublicKey: (
-    publicKey: string | undefined,
-    email: string | undefined
-  ) => Promise<void>;
-  verifyPGPPublicKey: (email: string | undefined) => Promise<void>;
+  // publishPGPPublicKey: (
```
Comment (xcjlo985) [inline app/src/hooks/usePpg.ts:20]: Remove all commented out code

File: app/src/hooks/usePpg.ts [hunk 3]
```diff
@@ -82,68 +77,61 @@ const usePgp = (): Hooks => {
...
+    api
+      .uploadPublicKey({
+        email: ethers.utils.hashMessage(email),
+        publicKeyArmored: publicKey
+      })
+      .then((result) => {
+        if (result) {
+          Alert.alert(
+            AlertTitle.Success,
+            `Your PGP key has been created with the password ${password}, and uploaded,please check your email`
```
Comment (bitcoinbrisbane) [inline app/src/hooks/usePpg.ts:94]:
```suggestion
            `Your PGP key has been created with the password ${password} and uploaded, please check your email to confirm.`
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Few minor issues on the alert text.

**Review by xiangzhe0812 — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**